src/components/Comments.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect } from "react";
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+import Timestamp from "react-timestamp";
+import { getCommentsByArticle } from "../utils/api";
+
+
+
+export default function Comments(){
+const[comments,setComments]=useState([])
+const {article_id}=useParams()
+
+
+
+
+useEffect(()=>{
+    getCommentsByArticle(article_id).then((allComments)=>{
+    setComments(allComments)
+    })
+},[])
+
```

Comment:
It would be good to consider what will be rendered if there are no comments on an article